react/OrderBy.js

```diff
@@ -64,36 +65,19 @@ class OrderBy extends Component {
   }
 
   render() {
-    const { orderBy, getLinkProps, runtime: { hints: { mobile } }, runtime } = this.props
```

Comment:
This Component does not hold state. You could change it to a functional component and use the `useRuntime` hook.